Documentation/.pages

```diff
@@ -6,3 +6,15 @@ nav:
     - Troubleshooting
     - Upgrade
     - Contributing
+    - Getting Started:
```

Comment:
Why is this section needed? 
1. Isn't this entry duplicated from line 2 above? 
2. Isn't this whole section a duplicate of the Getting-Started/.pages file?

Reply:
Without this section, the landing page will only show the above section of tabs under the title/header and the left side navigation will not list any tabs, which makes the page look emptyish and different from the current view.

  1. Even though this
  2.  Yes it is. But this whole section is added to keep the appearance of the landing page same as the current one and I think it is better to show those important tabs in the landing page itself.

Reply:
Got it, so we need to keep this here. But it still seems strange that "Getting started" is on both lines 2 and 9. Can we move this section up to replace line 2?

Reply:
Unfortunately moving these to replace line 2 doesn't seem to work. The way it works is, all the lines above these are fixed as a banner under the header of the page (not sure how it was acheived). And it is constant even if you swith between tabs/pages.
The ones below this works as a seperate section on the left side navigation of the page, which looks more appropriate too. One thing we could do is, we can change the "Getting Started" heading of this section to something else for example "Quick Links". But this would make it different than the other pages, where the page heading will be displayed on the top of the other pages left nav bar, but for getting started page the new name would appear.

Reply:
Ok, thanks for the confirmation of the behavior. Let's go with this for now, and we can follow up with another PR if there are other suggestions.